tools/UpdatePaths/Scripts/67761_floorvaredits_to_mappinghelpers.txt

```diff
@@ -0,0 +1,52 @@
+/turf/open/floor/mineral/@SUBTYPES{icon_state="bananium_dam"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam1"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam2"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam3"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam4"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam5"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam1_old"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam2_old"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam3_old"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam4_old"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam5_old"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="plastitanium_dam1"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="plastitanium_dam2"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="plastitanium_dam3"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="plastitanium_dam4"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
+/turf/open/floor/mineral/@SUBTYPES{icon_state="plastitanium_dam5"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
```

Comment:
```suggestion
/turf/open/floor/iron/@SUBTYPES{icon_state="damaged1"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/iron/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/iron/@SUBTYPES{icon_state="damaged2"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/iron/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/iron/@SUBTYPES{icon_state="damaged3"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/iron/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/iron/@SUBTYPES{icon_state="damaged4"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/iron/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/iron/@SUBTYPES{icon_state="damaged5"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/iron/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/iron/@SUBTYPES{icon_state="floorscorched1"} : /obj/effect/mapping_helpers/burnt_floor , /turf/open/floor/iron/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/iron/@SUBTYPES{icon_state="floorscorched2"} : /obj/effect/mapping_helpers/burnt_floor , /turf/open/floor/iron/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}

/turf/open/floor/plating/@SUBTYPES{icon_state="platingdmg1"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/plating/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/plating/@SUBTYPES{icon_state="platingdmg2"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/plating/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/plating/@SUBTYPES{icon_state="platingdmg3"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/plating/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/plating/@SUBTYPES{icon_state="panelscorched"} : /obj/effect/mapping_helpers/burnt_floor , /turf/open/floor/plating/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}

/turf/open/floor/wood/@SUBTYPES{icon_state="wood-broken"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood-broken2"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood-broken3"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood-broken4"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood-broken5"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood-broken6"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood-broken7"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}

/turf/open/floor/wood/@SUBTYPES{icon_state="wood_parquet-broken"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood_parquet-broken2"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood_parquet-broken3"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood_parquet-broken4"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood_parquet-broken5"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood_parquet-broken6"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood_parquet-broken7"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}

/turf/open/floor/wood/@SUBTYPES{icon_state="wood_tile-broken"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood_tile-broken2"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood_tile-broken3"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}

/turf/open/floor/wood/@SUBTYPES{icon_state="wood_large-broken"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood_large-broken2"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/wood/@SUBTYPES{icon_state="wood_large-broken3"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/wood/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}

/turf/open/floor/mineral/@SUBTYPES{icon_state="silver_dam"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}

/turf/open/floor/mineral/@SUBTYPES{icon_state="gold_dam"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}

/turf/open/floor/mineral/@SUBTYPES{icon_state="diamond_dam"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}

/turf/open/floor/mineral/@SUBTYPES{icon_state="plasma_dam"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}

/turf/open/floor/mineral/@SUBTYPES{icon_state="uranium_dam"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}

/turf/open/floor/mineral/@SUBTYPES{icon_state="bananium_dam"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}

/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam1"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam2"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam3"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam4"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam5"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}

/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam1_old"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam2_old"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam3_old"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam4_old"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/mineral/@SUBTYPES{icon_state="titanium_dam5_old"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}

/turf/open/floor/mineral/@SUBTYPES{icon_state="plastitanium_dam1"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/mineral/@SUBTYPES{icon_state="plastitanium_dam2"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/mineral/@SUBTYPES{icon_state="plastitanium_dam3"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/mineral/@SUBTYPES{icon_state="plastitanium_dam4"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
/turf/open/floor/mineral/@SUBTYPES{icon_state="plastitanium_dam5"} : /obj/effect/mapping_helpers/broken_floor , /turf/open/floor/mineral/@SUBTYPES{@OLD;icon_state=@SKIP;broken=@SKIP}
```

This will help readability whenever some poor sap wants to review what's going on.